src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -31,13 +31,17 @@
 
     public boolean scan_result;
     public String error_message;
-    //playerHitpoints to count how many hits player has taken, when it equals to 14, computer wins
-    //computerHitpoints to count how many hits AI has taken, when it equals to 14, player wins
+    //playershipsank = 5, player lose
+    //computershipsank = 5, player win
     public int playershipsank = 0;
     public int computershipsank = 0;
     public String AI_win = "You lose...T_T";
     public String Player_win = "You WIN!!! ^_^";
 
+    public boolean ezmode = false;
```

Comment:
I didn't notice this before I accepted, but I think this should be set to true because the default will be easy mode and changed to false when we the hard mode button is selected